### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by petrjasek**
- remove autogenerated docs for now - it made live preview slooow
- add initial architecture docs describing main components
- add docs contributing info - how to write docs for superdesk

see http://superdesk.readthedocs.io/en/docs/

### CODE REVIEW

**Comment by naspeh** [comment]
this commit "remove autogenerated docs" would be good to merge separately, it's no way to look at diff with it :)

**Comment by petrjasek** [comment]
created it here - https://github.com/superdesk/superdesk-core/pull/633, when merged will rebase this

**Comment by naspeh** [comment]
Now it's better, do you need a review here? or are you still working on this?

**Comment by petrjasek** [comment]
pls review

File: docs/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    $ make livehtml
+
+This will build docs and run a server on ``localhost:8000``. It will refresh as you modify documentation, but not when you modify python docstrings, in order to see some changes done there you still have to make some changes in rst files.
+
+Once you make a PR and it gets merged, you will see updated docs on `superdesk.readthedocs.io <http://superdesk.readthedocs.io/>`_.
+
+Updating docs
+^^^^^^^^^^^^^
+Documentation should be added/updated together with code changes in a single PR to `superdesk-core <https://github.com/superdesk/superdesk-core>` repository. There can be also PRs with only documentation.
```
Comment (naspeh) [inline docs/contributing.rst]: Forgotten underscore in the end of link

File: docs/architecture.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Architecture
+============
+Here there is info about main components in Superdesk and how these interact.
+To run superdesk we use `honcho <https://github.com/nickstenning/honcho>`_ to define
+processes for each of components:
```
Comment (naspeh) [inline docs/architecture.rst]: You could use [include directive](http://docutils.sourceforge.net/docs/ref/rst/directives.html#including-an-external-document-fragment) here, if not need wrap next strings as code.

Comment (petrjasek) [inline docs/architecture.rst]: yep, I would do it, but there is no `Procfile` in core repo..

Comment (petrjasek) [inline docs/architecture.rst]: yep, I would do it, but there is no Procfile in core repo..

Comment (naspeh) [inline docs/architecture.rst]: I see!

Comment (petrjasek) [inline docs/architecture.rst]: ah right

**Comment by naspeh** [comment]
:+1:

**Pull request merged**